R/runMDS.R

```diff
@@ -44,15 +47,23 @@
 NULL
 
 #' @importFrom stats cmdscale dist
-.calculate_mds <- function(x, ncomponents = 2, ntop = 500, subset_row = NULL, 
-    scale=FALSE, transposed=FALSE, method = "euclidean")
+.calculate_mds <- function(x, FUN = stats::dist, ncomponents = 2,
```

Comment:
Is there a non-**stats** `dist` you're concerned about?

Reply:
Not really, but I wouldn't bet on it.

Reply:
Package namespacing should protect you against other symbols in the search path.